LabDrivers/SIM900.py

```diff
@@ -352,7 +365,7 @@ def get_voltage(self, channel):
 
     def set_voltage(self, channel, voltage):
         channel = str(channel)
```

Comment:
I would add a docstring to this function to explain why 3 decimal is important and a potential reference to the user manual :)

Reply:
I added the docstring, however this function simply rounds it so that any sweep the user wishes to do will complete as expected and not bug out the SIM928. On the device itself, you can only modify 3 decimal places. As well, it can only process voltage values with at most 3 decimal places.